assets/scss/app.scss

```diff
@@ -14,3 +14,267 @@ html {
+
+    }
+    a:nth-of-type(1) {
+        color: $border-hover;
+        border: 1px solid $border-color;
+        border-radius: 3px;
+        float: left;
+
+        span:after {
+            float: right;
+            position: relative;
+            top: 6px;
+            right: -5px;
+            content: "";
+            width: 0;
+            height: 0;
+            border-style: solid;
+            border-width: 6px 4px 0 4px;
+            border-color: $border-color transparent transparent transparent;
+        }
+    }
+    a:nth-of-type(1):hover {
+        border-color: $border-hover;
+    }
+    .search {
+        float: left;
+        margin-right: 1rem;
+
+        input {
+            margin: 0;
+            background: $background-color;
+            padding: 10px;
+            border: 1px solid $border-color;
+            border-radius: 3px;
+            font-size: 1em;
+        }
+        input:hover {
+            border-color: $border-hover;
+        }
+        input:focus {
+            border-color: $etsy-blue;
+            background: white;
+            outline: none;
+        }
+        button {
+            background: $etsy-blue;
+            color: white;
+            font-weight: bold;
+            border: 1px solid darken($etsy-blue, 5%);
+            margin: 0;
+            margin-left: -7px;
+            border-top-right-radius: 3px;
+            border-bottom-right-radius: 3px;
+            padding: 10px;
+            font-size: 1em;
+        }
+        button:hover {
+            background: lighten($etsy-blue, 2%)
+        }
+        button:focus {
+            outline: none;
+        }
+    }
+    .account {
+        float: right;
+        a {
+            float: left;
+            color: $etsy-blue;
+            border: none;
+        }
+        a:nth-of-type(2) {
+            margin-left: 1rem;
+            border-radius: 3px;
+            border: 1px solid $etsy-blue;
+        }
+        a:nth-of-type(2):hover {
+            background: $etsy-blue-hover;
+        }
+        a:last-of-type {
+            margin-left: 1rem;
+            color: $border-hover;
+        }
+    }
+}
+.sidebar {
+    font-size: 0.8em;
+    p {
+        margin: 10px 0;
+        font-weight: bold;
+        color: $border-hover;
+    }
+    .bar {
+        background: white;
+        border: 1px solid $border-color;
+        border-radius: 3px;
+        padding: 12px;
+        li {
+            margin-bottom: 10px;
+            color: $border-hover;
+            font-weight: bold;
+        }
+        li a{
+            font-weight: normal;
+            text-decoration: none;
+            color: $etsy-blue;
+        }
+        li a:hover{
+            text-decoration: underline;
+        }
+        ul ul {
+            margin: 10px 0 0 20px;
+        }
+    }
+}
+article {
+    @include span-columns(4);
+    @include omega(3n);
+    border: 1px solid $border-color;
+    border-radius: 3px;
+    max-width: 225px;
+    margin: 5px;
```

Comment:
Try putting this margin before the `span-columns` since this will override the `margin-right` that the mixin includes